src/main/resources/application.yml

```diff
@@ -50,7 +50,7 @@ spring:
       iexBasePath: http://localhost:${wiremock.server.port}/
       iexToken: DUMMY_TOKEN
       iexValidRanges: max,5y,2y,1y,ytd,6m,3m,1m,1mm,5d,5dm,date,dynamic
-
+      historicalPricesCachePath: C:\Users\jamie\FUSE_historical_prices_cache
```

Comment:
See if using just a relative path would work here, in order to make it more compatible for other developers and also for deployment onto some remote server.